### ISSUE TITLE
2025-01-23, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#60]
# Today's Arxiv Papers

(1) [VideoLLaMA 3: Frontier Multimodal Foundation Models for Image and Video Understanding](http://arxiv.org/abs/2501.13106v1), 2025-01-22

(2) [Refining Input Guardrails: Enhancing LLM-as-a-Judge Efficiency Through Chain-of-Thought Fine-Tuning and Alignment](http://arxiv.org/abs/2501.13080v1), 2025-01-22

(3) [Does Table Source Matter? Benchmarking and Improving Multimodal Scientific Table Understanding and Reasoning](http://arxiv.org/abs/2501.13042v1), 2025-01-22

(4) [MONA: Myopic Optimization with Non-myopic Approval Can Mitigate Multi-step Reward Hacking](http://arxiv.org/abs/2501.13011v1), 2025-01-22

(5) [Pairwise RM: Perform Best-of-N Sampling with Knockout Tournament](http://arxiv.org/abs/2501.13007v1), 2025-01-22

(6) [Large Language Model-Based Semantic Communication System for Image Transmission](http://arxiv.org/abs/2501.12988v1), 2025-01-22

(7) [LLM4WM: Adapting LLM for Wireless Multi-Tasking](http://arxiv.org/abs/2501.12983v1), 2025-01-22

(8) [Implicit Causality-biases in humans and LLMs as a tool for benchmarking LLM discourse capabilities](http://arxiv.org/abs/2501.12980v1), 2025-01-22

(9) [OnionEval: An Unified Evaluation of Fact-conflicting Hallucination for Small-Large Language Models](http://arxiv.org/abs/2501.12975v1), 2025-01-22

(10) [Accessible Smart Contracts Verification: Synthesizing Formal Models with Tamed LLMs](http://arxiv.org/abs/2501.12972v1), 2025-01-22

(11) [It's complicated. The relationship of algorithmic fairness and non-discrimination regulations in the EU AI Act](http://arxiv.org/abs/2501.12962v1), 2025-01-22

(12) [Efficient Prompt Compression with Evaluator Heads for Long-Context Transformer Inference](http://arxiv.org/abs/2501.12959v1), 2025-01-22

(13) [GANQ: GPU-Adaptive Non-Uniform Quantization for Large Language Models](http://arxiv.org/abs/2501.12956v1), 2025-01-22

(14) [DeepSeek-R1: Incentivizing Reasoning Capability in LLMs via Reinforcement Learning](http://arxiv.org/abs/2501.12948v1), 2025-01-22

(15) [Correctness Assessment of Code Generated by Large Language Models Using Internal Representations](http://arxiv.org/abs/2501.12934v1), 2025-01-22

(16) [FilmAgent: A Multi-Agent Framework for End-to-End Film Automation in Virtual 3D Spaces](http://arxiv.org/abs/2501.12909v1), 2025-01-22

(17) [A Functional Software Reference Architecture for LLM-Integrated Systems](http://arxiv.org/abs/2501.12904v1), 2025-01-22

(18) [Architectural Fusion Through Contextual Partitioning in Large Language Models: A Novel Approach to Parameterized Knowledge Integration](http://arxiv.org/abs/2501.12901v1), 2025-01-22

(19) [Test-Time Preference Optimization: On-the-Fly Alignment via Iterative Textual Feedback](http://arxiv.org/abs/2501.12895v1), 2025-01-22

(20) [Generative AI Misuse Potential in Cyber Security Education: A Case Study of a UK Degree Program](http://arxiv.org/abs/2501.12883v1), 2025-01-22

(21) [WisdomBot: Tuning Large Language Models with Artificial Intelligence Knowledge](http://arxiv.org/abs/2501.12877v1), 2025-01-22

(22) [Mutation-Guided LLM-based Test Generation at Meta](http://arxiv.org/abs/2501.12862v1), 2025-01-22

(23) [ACEBench: Who Wins the Match Point in Tool Learning?](http://arxiv.org/abs/2501.12851v1), 2025-01-22

(24) [Adaptive Retrieval Without Self-Knowledge? Bringing Uncertainty Back Home](http://arxiv.org/abs/2501.12835v1), 2025-01-22

(25) [Open or Closed LLM for Lesser-Resourced Languages? Lessons from Greek](http://arxiv.org/abs/2501.12826v1), 2025-01-22

(26) [Revisit Self-Debugging with Self-Generated Tests for Code Generation](http://arxiv.org/abs/2501.12793v1), 2025-01-22

(27) [Generating Diverse Q&A Benchmarks for RAG Evaluation with DataMorgana](http://arxiv.org/abs/2501.12789v1), 2025-01-22

(28) [LLMs as Repositories of Factual Knowledge: Limitations and Solutions](http://arxiv.org/abs/2501.12774v1), 2025-01-22

(29) [NExtLong: Toward Effective Long-Context Training without Long Documents](http://arxiv.org/abs/2501.12766v1), 2025-01-22

(30) [EvidenceMap: Unleashing the Power of Small Language Models with Evidence Analysis for Biomedical Question Answering](http://arxiv.org/abs/2501.12746v1), 2025-01-22

(31) [Online Preference Alignment for Language Models via Count-based Exploration](http://arxiv.org/abs/2501.12735v1), 2025-01-22

(32) [A Call for Critically Rethinking and Reforming Data Analysis in Empirical Software Engineering](http://arxiv.org/abs/2501.12728v1), 2025-01-22

(33) [Paradigm-Based Automatic HDL Code Generation Using LLMs](http://arxiv.org/abs/2501.12702v1), 2025-01-22

(34) [Training Dialogue Systems by AI Feedback for Improving Overall Dialogue Impression](http://arxiv.org/abs/2501.12698v1), 2025-01-22

(35) [Combining Knowledge Graph and LLMs for Enhanced Zero-shot Visual Question Answering](http://arxiv.org/abs/2501.12697v1), 2025-01-22

(36) [EchoLM: Accelerating LLM Serving with Real-time Knowledge Distillation](http://arxiv.org/abs/2501.12689v1), 2025-01-22

(37) [Training Data Attribution (TDA): Examining Its Adoption & Use Cases](http://arxiv.org/abs/2501.12642v1), 2025-01-22

(38) [SoMa: Identifying, Exploring, and Understanding the DRAM Communication Scheduling Space for DNN Accelerators](http://arxiv.org/abs/2501.12634v1), 2025-01-22

(39) [Distillation Quantification for Large Language Models](http://arxiv.org/abs/2501.12619v1), 2025-01-22

(40) [Deep Learning-Based Identification of Inconsistent Method Names: How Far Are We?](http://dx.doi.org/10.1007/s10664-024-10592-z), 2025-01-22

(41) [Kimi k1.5: Scaling Reinforcement Learning with LLMs](http://arxiv.org/abs/2501.12599v1), 2025-01-22

(42) [Leveraging LLMs to Create a Haptic Devices' Recommendation System](http://arxiv.org/abs/2501.12573v1), 2025-01-22

(43) [O1-Pruner: Length-Harmonizing Fine-Tuning for O1-Like Reasoning Pruning](http://arxiv.org/abs/2501.12570v1), 2025-01-22

(44) [Understanding the LLM-ification of CHI: Unpacking the Impact of LLMs at CHI through a Systematic Literature Review](http://arxiv.org/abs/2501.12557v1), 2025-01-22

(45) [Human-like conceptual representations emerge from language prediction](http://arxiv.org/abs/2501.12547v1), 2025-01-21

(46) [An Empirically-grounded tool for Automatic Prompt Linting and Repair: A Case Study on Bias, Vulnerability, and Optimization in Developer Prompts](http://arxiv.org/abs/2501.12521v1), 2025-01-21

(47) [A Domain Adaptation Framework for Speech Recognition Systems with Only Synthetic data](http://arxiv.org/abs/2501.12501v1), 2025-01-21

(48) [The Journey Matters: Average Parameter Count over Pre-training Unifies Sparse and Dense Scaling Laws](http://arxiv.org/abs/2501.12486v1), 2025-01-21

(49) [An Empirical Characterization of Outages and Incidents in Public Services for Large Language Models](http://arxiv.org/abs/2501.12469v1), 2025-01-21

(50) [Adaptive PII Mitigation Framework for Large Language Models](http://arxiv.org/abs/2501.12465v1), 2025-01-21

(51) [Empowering AIOps: Leveraging Large Language Models for IT Operations ManagementOperations Management](http://arxiv.org/abs/2501.12461v1), 2025-01-21

(52) [Deploying Privacy Guardrails for LLMs: A Comparative Analysis of Real-World Applications](http://arxiv.org/abs/2501.12456v1), 2025-01-21

(53) [InternVideo2.5: Empowering Video MLLMs with Long and Rich Context Modeling](http://arxiv.org/abs/2501.12386v2), 2025-01-21

(54) [Expertise elevates AI usage: experimental evidence comparing laypeople and professional artists](http://arxiv.org/abs/2501.12374v1), 2025-01-21

(55) [Is Long Context All You Need? Leveraging LLM's Extended Context for NL2SQL](http://arxiv.org/abs/2501.12372v1), 2025-01-21

(56) [Treefix: Enabling Execution with a Tree of Prefixes](http://arxiv.org/abs/2501.12339v1), 2025-01-21

(57) [Automatic Labelling with Open-source LLMs using Dynamic Label Schema Integration](http://arxiv.org/abs/2501.12332v1), 2025-01-21

(58) [VARGPT: Unified Understanding and Generation in a Visual Autoregressive Multimodal Large Language Model](http://arxiv.org/abs/2501.12327v1), 2025-01-21

(59) [LLM-Assisted Knowledge Graph Completion for Curriculum and Domain Modelling in Personalized Higher Education Recommendations](http://arxiv.org/abs/2501.12300v1), 2025-01-21

(60) [MoGERNN: An Inductive Traffic Predictor for Unobserved Locations in Dynamic Sensing Networks](http://arxiv.org/abs/2501.12281v1), 2025-01-21

(61) [Divide-Then-Aggregate: An Efficient Tool Learning Method via Parallel Tool Invocation](http://arxiv.org/abs/2501.12432v1), 2025-01-21

(62) [Condor: Enhance LLM Alignment with Knowledge-Driven Data Synthesis and Refinement](http://arxiv.org/abs/2501.12273v1), 2025-01-21

(63) [FOCUS: First Order Concentrated Updating Scheme](http://arxiv.org/abs/2501.12243v1), 2025-01-21

(64) [InsTALL: Context-aware Instructional Task Assistance with Multi-modal Large Language Models](http://arxiv.org/abs/2501.12231v1), 2025-01-21

(65) [CDW-CoT: Clustered Distance-Weighted Chain-of-Thoughts Reasoning](http://arxiv.org/abs/2501.12226v1), 2025-01-21

(66) [Leveraging Large Language Models for Realizing Truly Intelligent User Interfaces](http://dx.doi.org/10.1145/3613905.3650949), 2025-01-21

(67) [You Can't Eat Your Cake and Have It Too: The Performance Degradation of LLMs with Jailbreak Defense](http://arxiv.org/abs/2501.12210v1), 2025-01-21

(68) [BiMarker: Enhancing Text Watermark Detection for Large Language Models with Bipolar Watermarks](http://arxiv.org/abs/2501.12174v1), 2025-01-21

(69) [AdaServe: SLO-Customized LLM Serving with Fine-Grained Speculative Decoding](http://arxiv.org/abs/2501.12162v1), 2025-01-21

(70) [Contextualizing Recommendation Explanations with LLMs: A User Study](http://arxiv.org/abs/2501.12152v1), 2025-01-21

(71) [Improving Influence-based Instruction Tuning Data Selection for Balanced Learning of Diverse Capabilities](http://arxiv.org/abs/2501.12147v1), 2025-01-21

(72) [Do LLMs Provide Links to Code Similar to what they Generate? A Study with Gemini and Bing CoPilot](http://arxiv.org/abs/2501.12134v1), 2025-01-21

(73) [Evaluating Efficiency and Engagement in Scripted and LLM-Enhanced Human-Robot Interactions](http://arxiv.org/abs/2501.12128v1), 2025-01-21

(74) [Can open source large language models be used for tumor documentation in Germany? -- An evaluation on urological doctors' notes](http://arxiv.org/abs/2501.12106v1), 2025-01-21

(75) [Dissecting the NVIDIA Hopper Architecture through Microbenchmarking and Multiple Level Analysis](http://arxiv.org/abs/2501.12084v1), 2025-01-21

(76) [Phishing Awareness via Game-Based Learning](http://arxiv.org/abs/2501.12077v1), 2025-01-21

(77) [EDoRA: Efficient Weight-Decomposed Low-Rank Adaptation via Singular Value Decomposition](http://arxiv.org/abs/2501.12067v1), 2025-01-21

(78) [PINNsAgent: Automated PDE Surrogation with Large Language Models](http://arxiv.org/abs/2501.12053v1), 2025-01-21

(79) [FREYR: A Framework for Recognizing and Executing Your Requests](http://arxiv.org/abs/2501.12423v1), 2025-01-21

(80) [Harnessing Generative Pre-Trained Transformer for Datacenter Packet Trace Generation](http://arxiv.org/abs/2501.12033v1), 2025-01-21

(81) [Rate-Aware Learned Speech Compression](http://arxiv.org/abs/2501.11999v1), 2025-01-21

(82) [CroMe: Multimodal Fake News Detection using Cross-Modal Tri-Transformer and Metric Learning](http://arxiv.org/abs/2501.12422v1), 2025-01-21

(83) [Linear Feedback Control Systems for Iterative Prompt Optimization in Large Language Models](http://arxiv.org/abs/2501.11979v1), 2025-01-21

(84) [Leveraging Graph Structures and Large Language Models for End-to-End Synthetic Task-Oriented Dialogues](http://arxiv.org/abs/2501.11977v1), 2025-01-21

(85) [Bridging Visualization and Optimization: Multimodal Large Language Models on Graph-Structured Combinatorial Optimization](http://arxiv.org/abs/2501.11968v1), 2025-01-21

(86) [A Hybrid Attention Framework for Fake News Detection with Large Language Models](http://arxiv.org/abs/2501.11967v1), 2025-01-21

(87) [TAD-Bench: A Comprehensive Benchmark for Embedding-Based Text Anomaly Detection](http://arxiv.org/abs/2501.11960v1), 2025-01-21

(88) [Proverbs Run in Pairs: Evaluating Proverb Translation Capability of Large Language Model](http://arxiv.org/abs/2501.11953v1), 2025-01-21

(89) [Web vs. LLMs: An Empirical Study of Learning Behaviors of CS2 Students](http://arxiv.org/abs/2501.11935v2), 2025-01-21

(90) [ALoFTRAG: Automatic Local Fine Tuning for Retrieval Augmented Generation](http://arxiv.org/abs/2501.11929v1), 2025-01-21

(91) [Integrate Temporal Graph Learning into LLM-based Temporal Knowledge Graph Model](http://arxiv.org/abs/2501.11911v1), 2025-01-21

(92) [Panoramic Interests: Stylistic-Content Aware Personalized Headline Generation](http://arxiv.org/abs/2501.11900v1), 2025-01-21

(93) [Med-R$^2$: Crafting Trustworthy LLM Physicians through Retrieval and Reasoning of Evidence-Based Medicine](http://arxiv.org/abs/2501.11885v2), 2025-01-21

(94) [From Drafts to Answers: Unlocking LLM Potential via Aggregation Fine-Tuning](http://arxiv.org/abs/2501.11877v1), 2025-01-21

(95) [Demons in the Detail: On Implementing Load Balancing Loss for Training Specialized Mixture-of-Expert Models](http://arxiv.org/abs/2501.11873v1), 2025-01-21

(96) [LLM-Agents Driven Automated Simulation Testing and Analysis of small Uncrewed Aerial Systems](http://arxiv.org/abs/2501.11864v1), 2025-01-21

(97) [EmbodiedEval: Evaluate Multimodal LLMs as Embodied Agents](http://arxiv.org/abs/2501.11858v1), 2025-01-21

(98) [Network-informed Prompt Engineering against Organized Astroturf Campaigns under Extreme Class Imbalance](http://arxiv.org/abs/2501.11849v1), 2025-01-21

(99) [A Survey on Memory-Efficient Large-Scale Model Training in AI for Science](http://arxiv.org/abs/2501.11847v1), 2025-01-21

(100) [Large Language Models with Human-In-The-Loop Validation for Systematic Review Data Extraction](http://arxiv.org/abs/2501.11840v1), 2025-01-21
